core/templates/dev/head/pages/profile-page/profile-page.directive.html

```diff
@@ -129,3 +129,14 @@ <h2 class="oppia-profile-username-small-screen">
     </span>
   </md-card>
 </div>
+<style>
+  .oppia-profile-user-stat-text-large-screen {
+    font-size: 16px;
+  }
+  .oppia-user-bio-span-custom {
+    white-space: pre-wrap;
+  }
+  .stat-help-text-span {
+    font-size: 16px;
+  }
+</style>
```

Comment:
Add newline below

Reply:
I have added a new line.

Reply:
It's still there! Github notates this with this:
![image](https://user-images.githubusercontent.com/5758256/67136841-6838cf80-f1e1-11e9-8e4b-4ea11874d0b5.png)